api/src/test/java/org/openmrs/notification/MessageServiceTest.java

```diff
@@ -111,6 +113,24 @@ public void sendMessage_shouldSendMessage() throws MessageException {
 				fail();
 			}
 		}
+		Template template = new Template();
```

Comment:
please create a new test for the case you want to test like

```java
@Test
public void methodName_shouldXXX () {
}
```

each test method should only cover one test case, so that when it fails we exactly know which path in the code did fail.

please read https://wiki.openmrs.org/display/docs/Unit+Testing+Conventions
*  Unit Test Conventions points to code templates you could set up in your IDE which makes it easier to write new tests.